robot_smach_states/src/robot_smach_states/world_model.py

```diff
@@ -27,34 +28,42 @@ def look_at_segmentation_area(robot, entity, volume=None):
     # Determine the height of the head target
     # Start with a default
     pos = entity.pose.frame.p
-    look_at_point_z = 0.7
+    x = pos.x()
+    y = pos.y()
+    z = 0.7
```

Comment:
@jlunenburg, these x, y and z are overwritten later

Reply:
volume might not be in entity.volumes

Reply:
Than this should be in the 'else'

Reply:
Fixed in 0346baf9c7e833cf19707e6b4ebfbcf62e463dd8